### DISCUSSION TITLE
Calendar not support  Buddhist Era

### DISCUSSION THREAD

**Discussion by pinyo-rungoral** [primefaces/community#466] [PrimeReact]
Buddhist Era is a year numbering system that primarily used in mainland Southeast Asian countries of Cambodia, Laos, Myanmar and Thailand as well as in Sri Lanka and Chinese populations of Malaysia and Singapore for religious or official occasions

Currently Calendar support feature formatDateTime for custom display date format but not provide feature for customFormat year in drop down and other relate when select
[ref from dayjs](https://day.js.org/docs/en/plugin/buddhist-era)

**Comment by melloware** [comment]
@pinyo-rungoral I would open a GitHub ticket This seems like a bigger change to make this work.

**Comment by mertsincan** [comment]
Yes, we do not plan to do any work on this at the moment.

Best Regards,
